app/configurations/config.tampere.js

```diff
@@ -289,4 +290,13 @@ export default configMerger(walttiConfig, {
   },
 
   showTenWeeksOnRouteSchedule: true,
+
+  parkAndRide: {
+    showParkAndRide: true,
+    showParkAndRideForBikes: true,
+    parkAndRideMinZoom: 13,
+    pageContent: {
+      default: HSLParkAndRideUtils,
+    },
```

Comment:
I think these can probably be left out as I think these are inherited from the waltti config.